### PR TITLE
feat(tianmu): Improve the readbility of stonedb. (#11)

### PR DESCRIPTION
**Pull request by konghaiya**
[summary]
1 class member variable refactor：exproter directory; 
2 class member variable refactor：loader directory;

## Summary about this PR

Issue Number: close #issue_number_you_created

## Tests Check List

- [ ] Unit test
- [ ] Integration test
- [ ] Manual test (add detailed scripts or steps below)
- [ ] No code

## Changelog

- [ ] New Feature
- [ ] Bug Fix
- [x] Improvement
- [ ] Performance Improvement
- [ ] Build/Testing/CI/CD
- [ ] Documentation
- [ ] Not for changelog (changelog entry is not required)

## Documentation

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features

### CODE REVIEW

**Comment by konghaiya** [comment]
CR, PTAL, @RingsC  @hustjieke  @isredstar @duanfuxiang0

**Review by hustjieke — APPROVED** [review]
LGTM

File: storage/tianmu/loader/load_parser.cpp [hunk 1]
```diff
@@ -29,44 +29,44 @@ namespace Tianmu {
...
                        std::unique_ptr<system::Stream> &f)
-    : attrs(attrs),
-      start_time(types::RCDateTime::GetCurrent().GetInt64()),
-      ioparam(iop),
-      pack_size(packsize),
-      rejecter(packsize, iop.GetRejectFile(), iop.GetAbortOnCount(), iop.GetAbortOnThreshold()),
-      no_obj(attrs[0]->NumOfObj()) {
+    : attrs_(attrs),
+      start_time_(types::RCDateTime::GetCurrent().GetInt64()),
+      ioparam_(iop),
```
Comment (RingsC) [inline storage/tianmu/loader/load_parser.cpp]: as the style of `start_time` and `pack_size`.  suggest: `io_param_`. and `no_obj_` changes to `num_of_obj_`

File: storage/tianmu/loader/load_parser.h [hunk 2]
```diff
@@ -49,32 +49,32 @@ class LoadParser final {
...
-  int64_t no_obj = 0;
-  int64_t row_no = 0;
-  int64_t dup_no = 0;
+  const char *cur_ptr_;
+  const char *buf_end_;
+  const system::IOParameters &ioparam_;
+  uint pack_size_ = 0;
+  Rejecter rejecter_;
+  uint cur_row_ = 0;
+  int64_t no_obj_ = 0;
```
Comment (RingsC) [inline storage/tianmu/loader/load_parser.h]: Here, using `no_obj` to `num_of_obj_`.

Do you know the meaning of `row_no_` and `dup_no_` ? what are they used for?

File: storage/tianmu/loader/parsing_strategy.h [hunk 3]
```diff
@@ -37,28 +37,28 @@ class ParsingStrategy final {
...
-
-  kmp_next_t kmp_next_delimiter;
-  kmp_next_t kmp_next_enclose_delimiter;
-  kmp_next_t kmp_next_eol;
-  kmp_next_t kmp_next_enclose_eol;
-
-  CHARSET_INFO *cs_info;
-  std::vector<char> temp_buf;
+  std::vector<core::AttributeTypeInfo> attr_infos_;
+  bool prepared_;
```
Comment (RingsC) [inline storage/tianmu/loader/parsing_strategy.h:44]: Overall rules: using the unify naming style and meanful names.

**Review by RingsC — CHANGES_REQUESTED** [review]

File: storage/tianmu/loader/load_parser.cpp (hunk 1 above)
Comment (konghaiya) [reply storage/tianmu/loader/load_parser.cpp]: The modification has been completed

**Review by RingsC — APPROVED** [review]
LGTM

**Pull request merged**